frontend/packages/kubevirt-plugin/integration-tests/tests/models/wizard.ts

```diff
@@ -239,11 +236,11 @@ export class Wizard {
   }
 
   async processNetworkStep(data: VMBuilderData) {
-    const { networks, provisionSource, template } = data;
+    const { networks, provisionSource } = data;
     for (const resource of networks) {
       await this.addNIC(resource);
     }
-    if (provisionSource?.method === ProvisionSource.PXE && template === undefined) {
+    if (provisionSource?.method === ProvisionSourceItem.PXE) {
```

Comment:
plus here

Reply:
I used the code listed here:
```
async selectOperatingSystem(operatingSystem: string) {
    await selectItemFromDropdown(view.operatingSystemSelect, view.dropDownItem(operatingSystem));
  }
```
and it couldn't select OS when running VMWare wizard

Reply:
No need to use template here based on the actual test results.